src/main/java/com/puppycrawl/tools/checkstyle/utils/CodePointUtil.java

```diff
@@ -56,45 +56,19 @@ public static boolean hasWhitespaceBefore(int index, int... codePoints) {
     }
 
     /**
-     * Removes trailing whitespaces.
+     * Removes whitespaces.
      *
      * @param codePoints array of unicode code points
      * @return unicode code points array with trailing whitespaces removed
      */
-    public static int[] stripTrailing(int... codePoints) {
+    public static int[] stripWhitespace(int... codePoints) {
```

Comment:
The name is confusing. The logic still strips only trailing whitespaces. Why did you rename it?

Reply:
yes, you are right. this logic strips only trailing whitespace. I have change the name again